docs/index.md

```diff
@@ -23,13 +23,47 @@
 
 Ralph is a toolbox for your learning analytics, it can be used as a:
 
-- **[LRS](https://en.wikipedia.org/wiki/Learning_Record_Store)**, a HTTP API server to collect xAPI statements (learning events), following the [ADL LRS standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree)
-- **command-line interface** (CLI), to build data pipelines the UNIX-way™️,
-- **library**, to fetch learning events from various backends, (de)serialize or
+- **[LRS](./tutorials/lrs.md)**, an HTTP API server to collect xAPI statements (learning events), following the [ADL LRS standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree)
```

Comment:
broken link? (./tutorials/lrs.md)

Reply:
For now yes... I can remove it until lrs tutorial is added though!